client/js/admin.js

```diff
@@ -38,7 +39,86 @@ $(function() {
 			return false;
 		}
 	});
-})
+
+	$("#uploadTrigger").click(function(e) {
+		var form = $("#fileUpload")
+		form.off("change")
+		form.on("change", () => {
+			var XHR = new XMLHttpRequest()
+			XHR.open("POST", "/static/")
+			var fData = new FormData(form[0].form)
+			XHR.onload = function() {
+				var res = $(".upload-name.upload-item")
+				res.removeClass("disabled")
+				var out = JSON.parse(XHR.response)
+				res.text(out.path)
+			}
+			XHR.send(fData)
+		})
+		form.click()
```

Comment:
This feels wrong.  It might _work_, but that doesn't sit right with me.

Maybe `form.submit()`?